src/agent/border_agent.cpp

```diff
@@ -42,8 +42,8 @@
 #include <string.h>
 #include <unistd.h>
 
-#include <openthread/thread_ftd.h>
 #include <openthread/border_agent.h>
+#include <openthread/thread_ftd.h>
```

Comment:
does t his need to be sorted?

Reply:
It fails pretty check:
https://github.com/openthread/ot-br-posix/runs/2397393075?check_suite_focus=true

Reply:
weird, I thought `t`hread should be after `p`platform.